sales/send_quotations/invoice_and_deliver_to_different_address.rst

```diff
@@ -0,0 +1,48 @@
+===========================================
+Deliver and invoice to different addresses
+===========================================
+
+In Odoo you can configure different addresses for delivery and
+invoicing. This is key, not everyone will have the same delivery
+location as their invoice location.
+
+Activate the feature
+=====================
+
+Go to :menuselection:\`SALES --> Configuration → Settings\` and activate
+the \*Customer Addresses\* feature.
+
+.. image:: media/invoice_and_deliver_to_different_address01.png
+    :align: center 
+
+Add different addresses to a quotation or sales order
+======================================================
+
+If you select a customer with an invoice and delivery address set, Odoo
+will automatically use those. If there’s only one, Odoo will use that
```

Comment:
Replace the invalid character ``’`` by ``'``

Reply:
Fixed